src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/TrackerObjectManager.NativeAot.cs

```diff
@@ -0,0 +1,549 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using static System.Runtime.InteropServices.ComWrappers;
+
+namespace System.Runtime.InteropServices
+{
+    // Defined in windows.ui.xaml.hosting.referencetracker.h.
+    internal enum XAML_REFERENCETRACKER_DISCONNECT
+    {
+        // Indicates the disconnect is during a suspend and a GC can be trigger.
+        XAML_REFERENCETRACKER_DISCONNECT_SUSPEND = 0x00000001
+    };
+
+    internal struct HostServices
```

Comment:
```suggestion
    internal static class HostServices
```

I don't see this struct instantiated anywhere.